### PR TITLE
Adding functionality to wishlist

### PR DESCRIPTION
**Pull request by zoef-yt**
1. In this pr, I have added functionality of adding to wishlist. You can click on the heart icon to add it to the wishlist the functionality to see the wishlist item is still not there as I wanted to keep the pr small. 
2. Removed the category of games as there were very less games to have category and all games have the same category which is included in the products.
3. Added CSS to selected wishlist icon.
[Products List]()

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
clean code and proper naming  looks good to merge

**Review by amartya-rs — APPROVED** [review]
Code looks good to me.

File: src/components/products-page/product-page-components/product-page-body.jsx [hunk 1]
```diff
@@ -8,18 +8,36 @@ const ProductsPageBody = () => {
...
 			{products.loading ? (
 				<div>LOADING AWESOMENESS.........</div>
 			) : (
 				products.data.map((product) => {
-					const { id, image, image2, name, price, offerMessage, inStock } = product;
-					return id == null ? (
+					const { _id, image, image2, name, price, offerPrice, offerMessage, inStock, hasOffer } = product;
+					return _id == null ? (
+						// if products are empty then this div will show
 						<div key={'1'}>
```
Comment (santhoshpemmaka) [inline src/components/products-page/product-page-components/product-page-body.jsx:19]: Better to pass a unique key inside the map function. I think the function name is unique. So you can pass the key as name. key={name}

**Comment by santhoshpemmaka** [comment]
Code structure looking good 💯 . Good to merge 👍

**Review by santhoshpemmaka — APPROVED** [review]

File: src/components/products-page/product-page-components/product-page-body.jsx (hunk 1 above)
Comment (zoef-yt) [reply src/components/products-page/product-page-components/product-page-body.jsx:19]: Ok, will do that.

**Pull request merged**